docs/local.rst

```diff
@@ -9,8 +9,11 @@ used with any server that provides the same API as DynamoDB.
 PynamoDB has been tested with two DynamoDB compatible servers, `DynamoDB Local <http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Tools.DynamoDBLocal.html>`_
 and `dynalite <https://github.com/mhart/dynalite>`_.
 
-All you need to do is set the ``host`` attribute on your ``Model``'s ``Meta`` class to the hostname and port
-that your server is listening on.
+To use a local server, you need to set the ``host`` attribute on your ``Model``'s ``Meta`` class to the hostname and port
+that your server is listening on. If you are using DynamoDB Local and also use ``rate_limited_scan`` on your models, you must also
```

Comment:
Small thing, can you use a `note` directive for the new paragraph? See https://raw.githubusercontent.com/rtfd/readthedocs.org/master/docs/features/embed.rst for an example